Assets/Scripts/Enemy/Enemy_Base.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+
+
+
+
+
+public class Enemy_Base : MonoBehaviour
+{
+    public float health;
+    public float damage;
+    public float moveSpeed;
+    public float jellyLevel;
+    public float coolDown;
+    public GameObject projectile;
+    public List<GameObject> positions;
+    public Collider2D hitbox;
+
+
+
+
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        this.gameObject.transform.position = Vector3.Lerp(positions[0].transform.position,   
```

Comment:
Why does the enemy base class do the ping pong motion? This kind of behavior should belong in the child, since not all enemies will be moving.

Reply:
Made an init to a to a static position, as a our base update. Unless another position is specified